lms/templates/student_profile/badge.underscore

```diff
@@ -0,0 +1,15 @@
+<div class="badge-image-container">
+  <img class="badge" src="<%- image_url %>" alt=""/>
+</div>
+<div class="badge-details">
+  <div class="badge-name"><%- badge_class.display_name %></div>
+  <p class="badge-description"><%- badge_class.description %></p>
+  <div class="badge-date-stamp">
+    <%-
+      interpolate(
+        gettext('Earned %(created)s.'),
```

Comment:
This will be hard to translate, so add a translation comment describing where it is used.